src/models/core/ClientSdk.ts

```diff
@@ -0,0 +1,10 @@
+
+/**
+ * Additional agents to add to the Client-SDK HTTP header.
+ *
+ * @public
+ **/
+export type CustomClientSdk = Record<string, string> & { ANSWERS_CORE?: never };
+
+/** The Client-SDK including the ANSWERS_CORE agent. */
+export type ClientSdk = Record<string, string> & { ANSWERS_CORE: string };
```

Comment:
Can we write these as interfaces instead? In Typescript, interfaces are generally preferred because they are easier to extend.

Reply:
Sure, I can write them as interfaces instead. But, I think `CustomClientSdk` would have to be something like this, where the additional agents could be specified as `undefined` to allow for an optional `ANSWERS_CORE` property:
``` 
interface CustomClientSdk {
  [agent: string]: string | undefined,
  ANSWERS_CORE?: never
}
```

Reply:
In some cases defining a type as an interface adds no additional complexity. In those cases, I think we should always pick an interface. Sometimes, using interfaces introduces some complexity that wouldn't be present in types. In those cases, it can go either way depending on if the documentation or simplicity in the code is more important. I think I'm leaning towards using an interface, but up to you

Reply:
It didn't add too much complexity, so I changed them to interfaces